content/en/docs/concepts/extend-kubernetes/operator.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Kubernetes operators
+reviewers:
+- erictune
+- lavalamp
+- cheftako
+- chenopis
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+Operators are software extensions to Kubernetes that make use of third-party
+resources to manage applications and their components. Operators follow
+Kubernetes priciples, notably the [control loop](/docs/concepts/#kubernetes-control-plane).
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Motivation
+
+The Operator concept aims to capture the key aim of a human operator who
+is managing a service or set of services. Human operators who look after
+specific applications and services have deep knowledge of how the system
+ought to behave, how to deploy it, and how to react if there are problems.
+
+People who run workloads on Kubernetes often like to use automation to take
+care of repeatable tasks. The Operator pattern captures how you can write
+code to automate a task beyond what Kubernetes itself provides.
+
+## Operators in Kubernetes
+
+Kubernetes is designed for automation. Out of the box, you get lots of
+built-in automation from the core of Kubernetes. You can use Kubernetes
+to automate deploying and running workloads, *and* you can automate how
+Kubernetes does that.
+
+Kubernetes' {{< glossary_tooltip text="controllers" term_id="controller" >}}
+concept lets you extend the cluster's behaviour without modifying the code
+of Kubernetes itself.
+Operators are clients of the Kubernetes API that act as controllers for
+a [Custom Resource](/docs/concepts/api-extension/custom-resources/).
+
+## Deploying Operators
+
+The most common way to deploy Operators is to add them to your cluster
+running outside of the {{< glossary_tooltip text="control plane" term_id="control-plane" >}},
+much as you would run any containerized application.
+For example, you can run the controller in your cluster as a Deployment.
+
+Just as with any Controller, you can also run an Operator entirely outside
+of Kubernetes, and sometimes that makes sense. For example, you might want to
+do this if your Operator is involved with ensuring the availability of Nodes
+themselves.
+
+## Using an Operator {#using-operators}
+
+Once you have an Operator deployed, you'd use it by adding, modifying or
+deleting the kind of resource that the Operator uses. For example:
```

Comment:
Do you really `deploy` an `Operator`? Or do you deploy a CRD, Controller?
This seems confusing to me, unless I have completely misunderstood.

Reply:
I'm not sure what wording best reflects what people who already know this tech well would write.

Reply:
Seems okay after a second read, since the Operator [pattern] has been defined.